dags/run_dbt_cleaning_gpt.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.models import Connection
+from airflow.operators.bash import BashOperator
+from airflow.operators.docker_operator import DockerOperator
+from docker.types import Mount
+from airflow.operators.python import PythonOperator
+import logging
+
+def print_debut():
+    logging.info(f"sa path variable : {os.getenv('sapath')}")
+
+with DAG(
+    'run_dbt_cleaning',
+    description='Check if dataset exist and create if not',
+    start_date=datetime(2024, 7, 18),
+    schedule_interval='@daily',
+    catchup = False
+) as dag:
+    #print(f"sa path variable : {os.getenv('sapath')}")
+    run_dbt_docker = DockerOperator(
+        task_id='run_dbt_docker',
+        image='ghcr.io/dbt-labs/dbt-bigquery:1.8.2',
+        command='run',
```

Comment:
Is it the command the container execute ? Can we do something like : `dbt run -m models/silver/` in order to build only the silver and then we can add anothers task for build the gold ?

Reply:
we could do something like this without the "dbt" in front of the command since it's already automatically called by the docker image, but it would require some more testing